.eslintrc

```diff
@@ -42,33 +42,33 @@
   "rules": {
     "prettier/prettier": 2,
 
-    "flowtype/space-after-type-colon": [2, "always"],
-    "flowtype/space-before-type-colon": [2, "never"],
-    "flowtype/space-before-generic-bracket": [2, "never"],
-    "flowtype/union-intersection-spacing": [2, "always"],
+    "flowtype/space-after-type-colon": 0,
```

Comment:
Do we get the same result if we just remove these lines completely? If so, it makes it a lot easier to parse the file, so let's delete them all.

If instead they default to `1`, then yeah this is the way to go.

Reply:
Sounds good! Thanks for the context.